Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIScriptingCocoa.mm

```diff
@@ -57,6 +57,73 @@
 
 using namespace WebExtensionDynamicScripts;
 
+static NSString * const idKey = @"id";
+static NSString * const matchesKey = @"matches";
+static NSString * const excludeMatchesKey = @"excludeMatches";
+static NSString * const persistAcrossSessionsKey = @"persistAcrossSessions";
+static NSString * const jsKey = @"js";
+static NSString * const cssKey = @"css";
+static NSString * const cssOriginKey = @"cssOrigin";
+static NSString * const allFramesKey = @"allFrames";
+static NSString * const runAtKey = @"runAt";
+static NSString * const userValue = @"user";
+static NSString * const authorValue = @"author";
+
+static NSString * const worldKey = @"world";
+static NSString * const mainWorld = @"main";
+static NSString * const isolatedWorld = @"isolated";
+
+static Ref<JSON::Array> toWebAPI(const Vector<String>& items)
+{
+    Ref result = JSON::Array::create();
+
+    for (const auto& item : items)
+        result->pushString(item);
+
+    return result;
+}
+
+static Vector<Ref<JSON::Object>> toWebAPI(const Vector<WebExtensionRegisteredScriptParameters>& scripts)
```

Comment:
`toWebAPI` is not the right name here, it was unfortunate that it was reusing the one for the web process here in the UI process. This is more like `toJSONSerialization`, something to signal this is for storage.